.gitignore

```diff
@@ -130,3 +130,4 @@ dmypy.json
 # Pyre type checker
 .pyre/
 tempResults/
+experiments
```

Comment:
Why should experiments folder be added to ignore? Since it's renamed to examples

Reply:
It was not a pure renaming. I still have the experiments locally, but I didn't want them in the repo any longer.